src/connection/server_support.c

```diff
@@ -2645,6 +2645,11 @@ css_process_server_server_connect (SOCKET master_fd)
   switch (STATIC_CAST (cubcomm::server_server, request))
     {
     case cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER:
+      if (!chn.send_int (static_cast < int >(cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER)))
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NET_PAGESERVER_CONNECTION, 1, chn.get_channel_id ());
+	  break;
+	}
```

Comment:
- Disable indent
- The error message is not a good fit. The message says: "Error connecting with the page server at $host". This is an error of the page server connection with the active transaction server. Create a new message (e.g. "Error connecting with the active transaction server").

```suggestion
      // *INDENT-OFF*
      if (!chn.send_int (static_cast<int> (cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER)))
	{
	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_NET_PAGESERVER_CONNECTION, 1, chn.get_channel_id ());
	  break;
	}
      // *INDENT-ON*
```

Reply:
Done